bin/dotd

```diff
@@ -3,25 +3,8 @@
 require 'cgi'
 require 'logger'
 require 'tmpdir'
-
-# Utility to strip consistent leading whitespace from heredoc strings, allowing
-# you to format your code more readably.
-#
-# Usage:
-#   <<-DOC.unindent
-#     Some text or other that I want to start in column 1.
-#         An actually indented line.
-#     This line still unindented.
-#   DOC
-#
-# from http://stackoverflow.com/a/5638187/5000129
-class String
-  # Strip leading whitespace from each line that is the same as the
-  # amount of whitespace on the least-indented line of the string.
-  def unindent
-    gsub /^#{scan(/^[ \t]+/).min_by(&:length)}/, ''
-  end
-end
+require_relative '../lib/cdo/cdo_cli'
+include CdoCli
```

Comment:
At some point, I thought there was a wish to make this script not depend on the rest of our codebase. This refactor violates this wish. Do we care?